E-Commerce/readme.md

```diff
@@ -46,13 +46,100 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+SELECT * 
+FROM products WHERE product_name 
+LIKE '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+SELECT * 
+FROM 
+    product_availability 
+JOIN  
+    products ON product_availability.prod_id = products.id 
+WHERE 
+    unit_price > 100;
+
 - [ ] List the 5 most expensive products
+SELECT prod_id, product_name, unit_price
+FROM
+    product_availability
```

Comment:
Here in your results, there are actually only two products (Mobile Phone X and Javascript Book), because there are multiple suppliers for each product. One way to interpret the question could be to instead look at the most expensive supplier for each product and return the top 5 from that. You could try that as an extra challenge.